src/Catty/ViewController/Continue&New/MaintainObject/MaintainScript/ScriptCollectionViewController.m

```diff
@@ -1178,7 +1179,7 @@ - (void)updateBrickCellData:(id<BrickCellDataProtocol>)brickCellData withValue:(
                                         minInputLength:kMinNumOfMessageNameCharacters
                                         maxInputLength:kMaxNumOfMessageNameCharacters
                               invalidInputAlertMessage:kLocalizedMessageAlreadyExistsDescription
-                                         existingNames:[Util allMessagesForProject:self.object.project]];
+                                         existingNames:(NSArray*)[Util allMessagesForProject:self.object.project]];
```

Comment:
Do not cast the set, but make use of the built-in `array` instance property: https://developer.apple.com/documentation/foundation/nsorderedset/1411531-array